### ISSUE TITLE
Can't login

### ISSUE THREAD

**Issue by Tuhwin** [aqw/xbmc-gamepass#425]
Same username+password works for the website and I have reentered it 10x to see if it's on my end, and I've also checked the settings xml for the gamepass app and it has my correct login credentials in it, so I know it's entered correctly.

I've replaced my username with my_correct_username below.
The password="xxxxxxxxxxxx" is untouched. It does however show more Xs than I have characters in my password, which I don't know if it is the real cause of this or not.

16:14:23.339 T:2220   DEBUG: [pigskin]: Debugging enabled.
16:14:23.340 T:2220   DEBUG: [pigskin]: Python Version: 2.7.13 (default, Jul 13 2017, 20:56:54) [MSC v.1900 32 bit (Intel)]
16:14:23.340 T:2220   DEBUG: [plugin.video.nfl.gamepass-2018.08.23]: script starting
16:14:23.341 T:2220   ERROR: EXCEPTION: Dialog not created.
16:14:23.341 T:2220   DEBUG: [plugin.video.nfl.gamepass-2018.08.23]: Error closing busy dialog: Dialog not created.
16:14:23.341 T:2220   DEBUG: [pigskin]: Request URL: https://www.nflgamepass.com/api/user/oauth/token
16:14:23.341 T:2220   DEBUG: [pigskin]: Method: post
16:14:23.342 T:2220   DEBUG: [pigskin]: Payload: {'username': 'my_correct_username', 'password': 'xxxxxxxxxxxx', 'client_id': u'42cc360e-6fbb-4472-9437-2f088b8730de', 'grant_type': 'password'}
16:14:23.380 T:2220   DEBUG: [pigskin]: Response code: 403
16:14:23.380 T:2220   DEBUG: [pigskin]: Response: <HTML><HEAD>
                                            <TITLE>Access Denied</TITLE>
                                            </HEAD><BODY>
                                            <H1>Access Denied</H1>
                                             
                                            You don't have permission to access "http&#58;&#47;&#47;www&#46;nflgamepass&#46;com&#47;api&#47;user&#47;oauth&#47;token" on this server.<P>
                                            Reference&#32;&#35;18&#46;dac1702&#46;1565187267&#46;18e795d
                                            </BODY>
                                            </HTML>
16:14:23.381 T:2220   DEBUG: [pigskin]: Login Failed.
16:14:23.381 T:2220   DEBUG: [pigskin]: <HTML><HEAD>
                                            <TITLE>Access Denied</TITLE>
                                            </HEAD><BODY>
                                            <H1>Access Denied</H1>
                                             
                                            You don't have permission to access "http&#58;&#47;&#47;www&#46;nflgamepass&#46;com&#47;api&#47;user&#47;oauth&#47;token" on this server.<P>
                                            Reference&#32;&#35;18&#46;dac1702&#46;1565187267&#46;18e795d
                                            </BODY>
                                            </HTML>
16:14:23.391 T:3308   DEBUG: ------ Window Init (DialogConfirm.xml) ------
16:14:23.391 T:3308    INFO: Loading skin file: DialogConfirm.xml, load type: KEEP_IN_MEMORY
16:14:23.471 T:4080   DEBUG: CVideoDatabase::RunQuery took 59 ms for 227 items query: select * from episode_view  WHERE ((episode_view.playCount IS NULL OR episode_view.playCount < 1))
16:14:23.667 T:4080   DEBUG: CVideoDatabase::RunQuery took 2 ms for 0 items query: select * from musicvideo_view
16:14:23.692 T:4080   DEBUG: CMusicDatabase::GetAlbumsByWhere query: SELECT albumview.*, albumartistview.* FROM albumview JOIN albumartistview ON albumartistview.idalbum = albumview.idalbum  WHERE albumview.strReleaseType = 'album'
16:14:23.986 T:4080   DEBUG: CMusicDatabase::GetAlbumsByWhere - query took 294 ms
16:14:24.245 T:4080   DEBUG: Skin Widgets: Total time needed to request recent items queries: 0:00:01.558000
16:14:24.245 T:4080   DEBUG: Skin Widgets: Total time needed for all queries: 0:00:14.990000
16:14:24.425 T:3308   DEBUG: Keyboard: scancode: 0x1c, sym: 0x000d, unicode: 0x000d, modifier: 0x0
16:14:24.518 T:3308   DEBUG: CInputManager::OnKey: return (0xf00d) pressed, action is Select
16:14:24.518 T:3308   DEBUG: Keyboard: scancode: 0x1c, sym: 0x000d, unicode: 0x0000, modifier: 0x0
16:14:24.763 T:3308   DEBUG: ------ Window Deinit (DialogConfirm.xml) ------
16:14:24.773 T:2220    INFO: CPythonInvoker(10, C:\Users\Linus\AppData\Roaming\Kodi\addons\plugin.video.nfl.gamepass\default.py): script aborted
16:14:24.794 T:2220    INFO: Python script stopped

Full log also attached.
[kodi.log](https://github.com/aqw/xbmc-gamepass/files/3477534/kodi.log)

**Comment by kabukimurder** [comment]
Try Baardisks fork. It should be considered the only none-obsolete repo for this addon:
https://github.com/baardisk/xbmc-gamepass

**Comment by Tuhwin** [comment]
Thank you for that info. I can however not run Kodi18 on the machine without getting video/sound issues, and baardisk's fork only supports Leia, no?

**Comment by weordie** [comment]
Did you solve this? Because I've been having the same issue

**Comment by Tuhwin** [comment]
Nope. The Kodi18 version from baardisk works, but as Kodi18 itself doesn't work well on my system I'm back on Kodi17 and waiting for a fix from aqw, but no idea if there will ever be one.

**Comment by Ghoschdi** [comment]
I checked baardisk's changes and he literally just changed the required addons to the Leia versions. So grabbing the master from here https://github.com/aqw/xbmc-gamepass/archive/master.zip and installing it should work on 17.x

**Comment by weordie** [comment]
I keep getting a message saying it failed to install xbox.gui 5.12.0

**Comment by Tuhwin** [comment]
Then the fix in terms of login lies in the dependencies then, as this master doesn't work for me with K17. Can't login with it.

**Comment by jm-duke** [comment]
Doing some clean-up and this issue report is really old, so I'm closing it.
